### DISCUSSION TITLE
One click range pickup【Google Translate】

### DISCUSSION THREAD

**Discussion by dz20161011** [azerothcore/azerothcore-wotlk#14086] [Modules]
[One click range pickup.zip](https://github.com/azerothcore/azerothcore-wotlk/files/10167356/One.click.range.pickup.zip)
This is a mod that picks up items with one click range
The technology given to me by my friend is not enough and I don’t know how to apply it
Also tried googling
I hope capable friends can help
1. Increase the function switch of one-key pickup
2. Increase the category of one-click pickup currency equipment or sundries
3. These functions are best operated in command format

Thank you for your help
The above content is from Google Translate

**Comment by virgo77** [comment]
Not sure that RAR files inside a zip one and which are within chinese/japanese text format would be a "safe" mod.
If you want to contribute, create a Github account, then pull on a request about your mod. It will be checked and tested. If your mod is safe, maybe it will be included. But I don't know if this version client will accept a one-click pickup mod. If it was, some mods would be created years ago.
I hope it's not a malware as I will not test it. I'm not confident about it.
Bye
V.
